config/packages.cmake

```diff
@@ -231,6 +231,8 @@ if(ENABLE_PARTITIONING)
 
 endif()
 
+cinch_target_link_libraries(flecsi "${FLECSI_RUNTIME_LIBRARIES};${PARTITION_LIBRARIES};${IO_LIBRARIES}")
```

Comment:
If it is, that's not very nice and maybe I should change the interface in cinch.